server/core/src/https/views/admin/admin.rs

```diff
@@ -0,0 +1,45 @@
+use crate::https::extractors::{AccessInfo, VerifiedClientInformation};
+use crate::https::middleware::KOpId;
```

Comment:
Is this page needed? Seems like it doesn't do much?

Reply:
It's unused yes

Reply:
Should we remove it then? Just to help keep this a bit more manageable as a PR